src/guides/getting-started-with-drizzle-and-react/index.md

```diff
@@ -410,41 +456,6 @@ render() {
 
 In the next section, we will replace "Drizzle is ready" with an actual component that will read from the store. If you refresh your browser and run this app now, you should see "Loading Drizzle..." briefly flash on screen and then subsequently "Drizzle is ready".
 
-### Full component code
```

Comment:
I may be missing context since I'm looking at this in GitHub, but shouldn't we leave in the section with what the completed code should look like?

Reply:
The full source of the App.js is now higher in the tutorial under the heading 'Wire up the App component'. The issue that you reported noted the difference in the App.js between the code in the tutorial and the code that you had locally from running create-react-app. By default create-react-app creates a React functional component rather than the class shown in the tutorial. A class component is preferred for managing state and using life cycle methods. You can do it in a functional component but, there are some extra hoops to jump through first. The approach I took was to instruct the user to replace the content of App.js with the code presented in the tutorial - that is, the full source code for the component. The tutorial then explains the App.js code. We could add the full source for App.js at the end - but there is only a few additional lines. I thought that would be unnecessary as its pretty clear from the tutorial what's happening. However, I am happy to add it if you think it's worthwhile.

Reply:
Nope, that seems fine! I figured I was missing some context, thank you for clarifying!